OtripleS.Web.Api/Brokers/Storages/IStorageBroker.Meals.cs

```diff
@@ -0,0 +1,14 @@
+﻿// ---------------------------------------------------------------
+// Copyright (c) Coalition of the Good-Hearted Engineers
+// FREE TO USE AS LONG AS SOFTWARE FUNDS ARE DONATED TO THE POOR
+// ---------------------------------------------------------------
+
```

Comment:
Remove this empty line. Leave only one to separate the header and the code

Reply:
Done